packages/rainbowkit/src/locales/en_US.json

```diff
@@ -37,9 +37,11 @@
     "recent": "Recent",
     "status": {
       "opening": "Opening %{wallet}...",
+      "opening_mobile": "Continue in %{wallet}...",
       "not_installed": "%{wallet} is not installed",
       "not_available": "%{wallet} is not available",
-      "confirm": "Confirm connection in the extension"
+      "confirm": "Confirm connection in the extension",
+      "confirm_mobile": "Accept connection request in the wallet"
```

Comment:
Wanna make sure it's nice to include internalization if user are going to use this package with rainbowkit.